reference/docs-conceptual/dev-cross-plat/resolving-dependency-conflicts.md

```diff
@@ -0,0 +1,1097 @@
+  is being requested by our module. But this doesn't work for dependencies of dependencies; for example, if `CsvHelper` itself raised an `AssemblyResolve` event. There are other, harder ways to
+  do this, such as comparing requested assemblies to the metadata of assemblies in the
+  `Dependencies` directory. But, in this example, we've made this checking more explicit to both
+  highlight the issue and simplify the example.
+
+This is the key difference between the `LoadFile()` strategy and the ALC strategy: the
+`AssemblyResolve` event must service all loads in the Application Domain, making it much harder to
+keep our dependency resolution from being tangled with other modules. However, the lack of ALCs in
+.NET Framework makes this option worth understanding.
+
+### Custom Application Domains
+
+The final and most extreme option for assembly isolation is to use custom Application Domains.
+Application Domains (used in the plural) are only available in .NET Framework. They are used to
+provide in-process isolation between parts of a .NET application. One of the uses is to isolate
+assembly loads from each other within the same process.
+
+However, Application Domains are serialization boundaries. Objects in one Application Domain can't
+be referenced and used directly by objects in another Application Domain. You can work around this
+by implementing `MarshalByRefObject`. But when you don't control the types, as is often the case
+with dependencies, it's not possible to force an implementation here. The only solution is to make
+large architectural changes. The serialization boundary also has serious performance implications.
+
+Because Application Domains have this serious limitation, are complicated to implement, and only
+work in .NET Framework, we won't give an example of how you might use them here. While they're worth
+mentioning as a possibility, they not recommend.
+
+If you're interested in trying to use a custom Application Domain, the following links might help:
+
+- [Conceptual documentation on Application Domains](/dotnet/framework/app-domains/application-domains)
+- [Examples for using Application Domains](/dotnet/framework/app-domains/use)
+
+## Solutions for dependency conflicts that don't work for PowerShell
+
+Finally, we'll address some possibilities that come up when researching .NET dependency conflicts in
+.NET that can look promising but generally won't work for PowerShell.
+
+These solutions have the common theme that they are changes to deployment configurations for an
+environment where you control the application and possibly the entire machine. These solutions are
+oriented toward scenarios like web servers and other applications deployed to server environments,
+where the environment is intended to host the application and is free to be configured by the
+deploying user. They also tend to be very much .NET Framework focused, meaning they do not work
```

Comment:
```suggestion
deploying user. They also tend to be very much .NET Framework oriented, meaning they do not work
```
I think this oriented phrases it better. I could be wrong so feel free to ignore.